orangehrm-quick-start-guide.html

```diff
@@ -1,106 +1,111 @@
-             src='https://raw.githubusercontent.com/wiki/orangehrm/orangehrm/mobile/app_store_en_US.svg'/>
-    </a></li>
-</ul>
-
-</body>
-<script type="text/javascript" src="installer/guide/index.js"></script>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>OrangeHRM Quick Start Guide</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
+      a {
+        color: #009900;
+        text-decoration: underline;
+      }
+
+      #logo {
+        text-align: center;
+        height: 8rem;
+        margin: 0 auto;
+        background: url(logo.png) no-repeat;
+		background-size: contain;
+        background-position: center;
+      }
+
+      h1 {
+        text-align: center;
+        font-size: 24px;
+      }
+
+      h2 {
+        text-align: left;
+        font-size: 16px;
+      }
+
+      body {
+        font-family: Arial, Helvetica, sans-serif;
+        margin: 5px 10%;
+      }
+
+      li {
+        padding-bottom: 5px;
+        padding-top: 5px;
+      }
+
+      #content {
+        margin-left: 50px;
+        margin-right: 50px;
+      }
+
+      .version {
+        text-decoration: none;
+      }
+
+      .play-store-img {
+        height: 54px;
+        margin-bottom: -20px;
+      }
+
+      .app-store-img {
+        height: 41px;
+        padding: 9px;
+        margin-bottom: -24px;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="logo"></div>
+
+    <h1>Welcome to OrangeHRM Quick Start Guide!</h1>
+
+    <p>
+      Below you can find links to our web site where installation prerequisites
+      and installation process are explained. Make sure you are connected to the
+      Internet to follow these links. If you find any problem, feel free to post
+      it at our
+      <a href="http://www.orangehrm.com/forum/" target="_blank">forum</a>.
+    </p>
+
+    <h2>Prerequisites for installing OrangeHRM <ins class="version"></ins></h2>
+
+    <ul>
+      <li>
+        <a
+          href="https://starterhelp.orangehrm.com/hc/en-us/articles/6187572000540-Prerequisites-for-installing-OrangeHRM-Starter-in-Linux"
+          target="_blank"
+        >
+          Prerequisites for installing in Linux
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://starterhelp.orangehrm.com/hc/en-us/articles/6187576427804-Prerequisites-for-installing-OrangeHRM-Starter-in-Windows"
+          target="_blank"
+        >
+          Prerequisites for installing in Windows
+        </a>
+      </li>
+    </ul>
+
+    <h2>Installing OrangeHRM <ins class="version"></ins></h2>
+
+    <ul>
+      <li>
+        <a
+          href="https://starterhelp.orangehrm.com/hc/en-us/articles/5295915003666-OrangeHRM-Starter-Installation-Guide"
+          target="_blank"
+        >
+          Installing OrangeHRM using web installer
```

Comment:
Any idea to add upgrader link?
https://starterhelp.orangehrm.com/hc/en-us/articles/6937346912402-OrangeHRM-Starter-Upgrade-Guide-For-5x-versions-

Reply:
Not sure. btw this is quick start guide, upgrader is for advance users who have already running setup